### ISSUE TITLE
JSON roleInfo.js is not updated

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria#2234]
The JSON file available at https://github.com/w3c/aria/blob/main/common/script/roleInfo.js is outdated. 
For instance, the `aria-braillelabel` property cannot be utilized in:

> [caption](https://w3c.github.io/aria/#caption), [code](https://w3c.github.io/aria/#code), [definition](https://w3c.github.io/aria/#definition), [deletion](https://w3c.github.io/aria/#deletion), [emphasis](https://w3c.github.io/aria/#emphasis), [generic](https://w3c.github.io/aria/#generic), [insertion](https://w3c.github.io/aria/#insertion), [mark](https://w3c.github.io/aria/#mark), [none](https://w3c.github.io/aria/#none), [paragraph](https://w3c.github.io/aria/#paragraph), [strong](https://w3c.github.io/aria/#strong), [subscript](https://w3c.github.io/aria/#subscript), [suggestion](https://w3c.github.io/aria/#suggestion), [superscript](https://w3c.github.io/aria/#superscript), [term](https://w3c.github.io/aria/#term), [time](https://w3c.github.io/aria/#time)

(and `presentation`, which is a synonym of `none`), but the JSON file does not include this information.

Moreover, the array associated with the "allProps" key appears to be unnecessary.

**Comment by pkra** [comment]
I believe that file currently only gets updated on REC.

But, yes, there are several bugs. Some I've documented as part of my (dare I say "ongoing") work at https://github.com/w3c/aria-common/pull/106

**Comment by giacomo-petri** [comment]
+ `image` role is missing `parentRoles: ["section"] `

**Comment by pkra** [comment]
@giacomo-petri see https://github.com/w3c/aria/issues/2073 and https://github.com/w3c/aria/issues/2165

**Comment by pkra** [comment]
#2624 is proposing a manual update. #2625 an automatic one.
